TESTS/mbed_platform/FileHandle/main.cpp

```diff
@@ -353,7 +375,17 @@ void test_fprintf_fscanf()
     TEST_ASSERT_TRUE(TestFile<FS>::functionCalled(TestFile<FS>::fnWrite));
     TEST_ASSERT_TRUE(std::ferror(file) != 0);
     TEST_ASSERT_TRUE(fprintf_ret < 0);
-
+#else
+    // Writing remaining available file space of 2 characters
+    // to make further fscanf() test to pass.
+    // write 2; expected written 2
```

Comment:
```suggestion
// Add the EOF manually as it is not supported in MicroLib
```
Perhaps the above to replace lines 379-381

Reply:
Done.